runtests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+export FLASK_ENV="TESTING"
+export PYTHONPATH=$(pwd)  # setting project root as PYTHONPATH
```

Comment:
Use `$(dirname $0)` here, since `runtests.sh` could be called from another folder.